shell/browser/api/electron_api_web_contents.cc

```diff
@@ -3213,6 +3224,7 @@ void WebContents::IncrementCapturerCount(gin::Arguments* args) {
                     .Release();
 }
 
+// TODO(codebytere): remove in Electron v23.
 void WebContents::DecrementCapturerCount(gin::Arguments* args) {
```

Comment:
Given we are only deprecating the methods in Electron v22, shouldn't we keep the patch `patches/chromium/fix_expose_decrementcapturercount_in_web_contents_impl.patch` around for this implementation to work and only remove it in v23 along with these methods ?

Also, for v22 shouldn't these methods log a deprecation message ?